notebook/templates/page.html

```diff
@@ -41,6 +38,8 @@
             bootstraptour: 'components/bootstrap-tour/build/js/bootstrap-tour.min',
             'jquery-ui': 'components/jquery-ui/ui/minified/jquery-ui.min',
             moment: 'components/moment/min/moment-with-locales',
+            react: 'components/react/react.production.min',
```

Comment:
Is it possible to optionally use the development version of react if the notebook is built using the developer tools instructions?

Reply:
I wondered that too, didn't want to go too far down the rabbit hole in lieu of other yaks that wanted shaving.